source/docs/software/wpilib-tools/robot-simulation/introduction.rst

```diff
@@ -31,6 +31,11 @@ Ensure the :guilabel:`Desktop Development with C++` option is checked in the Vis
 .. image:: images/vs-build-tools.png
    :alt: Screenshot of the Visual Studio build tools option
 
+Enabling Desktop Support (Python)
+---------------------------------
+
+GUI simulation support is installed when you ``pip install robotpy``.
```

Comment:
```suggestion
GUI simulation support is installed when you install robotpy.
```

The install instructions use other pip commands. All the install instructions should be isolated to that page